### PR TITLE
Add hermit configuration to dev docs

### PR DESCRIPTION
**Pull request by romulets**
### Summary of your changes
Update dev-docs to explicitly mention `hermit` as part of development set up.

### Checklist
- [x] I have added the necessary README/documentation (if appropriate)

### CODE REVIEW

File: dev-docs/Development.md [hunk 1]
```diff
@@ -5,13 +5,29 @@
 For Golang, we try to follow [Google's code style](https://google.github.io/styleguide/go/)
 For Python we try to follow [PEP8](https://peps.python.org/pep-0008/) style guid
 
+### Hermit
+
+We use [hermit](https://cashapp.github.io/hermit/) to manage development environment.
```
Comment (orestisfl) [inline dev-docs/Development.md:10]:
```suggestion
We use [hermit](https://cashapp.github.io/hermit/) to manage the development environment.
```

File: dev-docs/Development.md [hunk 2]
```diff
@@ -5,13 +5,29 @@
...
+
+To install, [follow the official docs](https://cashapp.github.io/hermit/usage/get-started/#installing-hermit).
```
Comment (orestisfl) [inline dev-docs/Development.md:12]: I believe just running `source bin/activate-hermit` will do everything you need, no need to follow any instructions.

**Review by orestisfl — COMMENTED** [review]
There is a reference to hermit in the main README and `bin/README.hermit.md` as well, can you collect everything under one file?

File: dev-docs/Development.md (hunk 1 above)
Comment (romulets) [reply dev-docs/Development.md:10]: I'll collect everything under one file! I did not see the README.hermit.md. It actually covers it all. I'll just point to it

**Comment by romulets** [comment]
> There is a reference to hermit in the main README and `bin/README.hermit.md` as well, can you collect everything under one file?

I'll collect everything under one file! I did not see the README.hermit.md. It actually covers it all. I'll just point to it

File: dev-docs/Development.md (hunk 2 above)
Comment (romulets) [reply dev-docs/Development.md:12]: The source bin activates the hermit in the current folder/project. The zsh integration watches where you are and activates automatically the env.

It's nice once you are working with multiple terminal sessions at the same time (my case)

But it's already mentioned as recommended in the README.hermit.md

**Pull request merged**